traffic_ops/traffic_ops_golang/cdni/shared.go

```diff
@@ -154,18 +149,19 @@ func PutHostConfiguration(w http.ResponseWriter, r *http.Request) {
 		api.HandleErr(w, r, inf.Tx.Tx, http.StatusInternalServerError, nil, fmt.Errorf("getting async tx: %w", err))
 		return
 	}
+	logTx, err := db.Begin()
```

Comment:
I'm confused why we need different `asyncTx`, `logTx`, etc. Why can't we just use `inf.Tx.Tx`?

Reply:
I think its because of the way I wrote the async status endpoints way back when. since those hold the `defer tx.Commit` i think that requires all of these to be separate. i never really thought of it that way until right now so ill go in and fix it when i have time to test everywhere that uses it

Reply:
Oh yeah, it makes sense for when you have to hand off a DB transaction to a goroutine for it to be completed asynchronously, but I think when it's all handled synchronously all in the same goroutine it can use the same transaction.